app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
I wrote an event in Backbone.View for Settings page for .additional-visibility-icon class, but for some reason it doesn't get called on click on image tag with the class set. Instead I get in the javascript console:
15:34:51.812 An empty string can not be an argument of the method getElementById().1 jquery.js:2823:0


Could somebody please check what is wrong?
Here is my changes: http://pastebin.com/LGCt0Mft

Or at least point me what to check.